### PR TITLE
Update Metabase URLs to custom domains

### PR DESCRIPTION
**Pull request by ivanELEC**
## Description of change

[Add TLS cert and custom domain for metabase deployments in app store](https://dsdmoj.atlassian.net/browse/CRM457-1546)

### CODE REVIEW

**Review by jsugarman — APPROVED** [review]

**Comment by ivanELEC** [comment]
New URLs currently deploying to app store - will merge once app store merge is complete

**Pull request merged**
